include/engine/api/route_api.hpp

```diff
@@ -503,7 +503,17 @@ class RouteAPI : public BaseAPI
                 nodes.emplace_back(static_cast<uint64_t>(node_id));
             }
         }
+        std::vector<int64_t> ways;
```

Comment:
Are ways ID signed in OSM?

Reply:
@akashihi The OSM Way ID is unsigned, but Way IDs in the annotation are signed: the negative value means opposite moving direction, while positive means moving forward along the way.